recipes/coincbc/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+UNAME="$(uname)"
+export CFLAGS="-O3"
```

Comment:
Please never set CFLAGS and friends without grabbing their current value:

    export CFLAGS="${CFLAGS} -O3"